### PR TITLE
ref(pii): Added regex description

### PR DESCRIPTION
**Pull request by priscilawebdev**
closes: https://app.asana.com/0/1158284503473033/1179271443094538

. Added Regex Description in the list

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/dataPrivacyRulesForm/utils.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ function getRuleTypeLabel(type: RuleType) {
     case RuleType.IP:
       return t('IP addresses');
     case RuleType.PATTERN:
-      return t('Regex matches');
+      return t('Regex match');
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/dataPrivacyRulesForm/utils.tsx:32]:
```suggestion
      return t('Regex matches');
```

File: src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/rulesList.tsx [hunk 2]
```diff
@@ -1,96 +1,214 @@
...
+        {rules.map(({id, method, type, source, customRegularExpression}) => {
+          const className = `gridCell-${id}`;
+          const methodLabel = getMethodTypeLabel(method);
+          const typeLabel = getRuleTypeLabel(type);
+          const methodDescription =
+            type === RuleType.PATTERN ? customRegularExpression : typeLabel;
+
+          const gridCellProps = !disabled
+            ? {
+                onMouseOver: this.handleMouseOver(className),
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/rulesList.tsx:93]: Do you want mouseEnter here instead? the mouseover event is also triggered when children are hovered.

File: src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/rulesList.tsx [hunk 3]
```diff
@@ -1,96 +1,214 @@
...
       color: ${p.theme.gray400};
       background: ${p.theme.gray100};
   `}
+  ${p =>
+    p.hoveredClassname &&
+    `
+    .${p.hoveredClassname} {
+      background-color: ${p.theme.gray100};
+    }
+  `}
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/rulesList.tsx:191]: Would this be simpler to implement if the Grid was a table element and the css grid was applied to each row? I've also not see many tables in the application do row highlights on hover.

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/rulesList.tsx:191]: yes, this is a good idea... But sorry... I will prefer to leave as it is now

**Review by untitaker — APPROVED** [review]
Screenshot 2020-06-09 at 17 17 06
It seems that the content of the organization rules dropdown is now misaligned with the other rules. For that reason alone I would suggest to avoid putting more effort into this alignment and go back to the regular "it's just text" concept, but it's your call

**Review by markstory — APPROVED** [review]

**Review by untitaker — APPROVED** [review]

**Pull request merged**